src/app/pages/details/common/resource-box/resource-boxes.js

```diff
@@ -17,27 +17,41 @@ import useUserContext from '~/contexts/user';
 import linkhelper from '~/helpers/link';
 import cn from 'classnames';
 
-function CommonsHubBox({model}) {
+function CommonsHubBox() {
+    const {
+        communityResourceHeading: heading,
+        communityResourceLogoUrl: logoUrl,
+        communityResourceUrl: url,
+        communityResourceCta: cta,
+        communityResourceBlurb: blurb,
+        communityResourceFeatureLinkUrl: featureUrl,
+        communityResourceFeatureText: featureText
+    } = useDetailsContext();
+
+    if (!url) {
+        return null;
+    }
+
     return (
         <div className="resource-box double">
             <div className="top-line">
-                <h3>{model.heading}</h3>
-                <img src={model.logoUrl} alt="" />
+                <h3>{heading}</h3>
+                <img src={logoUrl} alt="" />
             </div>
             <div className="blurb">
```

Comment:
I see you're setting one of the urls to null below, should this code test for null and skip showing the img in that case?  I'm not 100% sure what <img> does with a null url, hopefully it doesn't show an ugly broken image 😬